substrate/client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -408,44 +463,53 @@ where
 		}
 
 		let operation_id = block_guard.operation().operation_id();
-		let event = self
-			.client
-			.executor()
-			.call(hash, &function, &call_parameters, CallContext::Offchain)
-			.map(|result| {
-				FollowEvent::<Block::Hash>::OperationCallDone(OperationCallDone {
-					operation_id: operation_id.clone(),
-					output: hex_string(&result),
-				})
-			})
-			.unwrap_or_else(|error| {
-				FollowEvent::<Block::Hash>::OperationError(OperationError {
-					operation_id: operation_id.clone(),
-					error: error.to_string(),
-				})
-			});
-
-		let (rp, rp_fut) = method_started_response(operation_id, None);
+		let client = self.client.clone();
 
+		let (rp, rp_fut) = method_started_response(operation_id.clone(), None);
 		let fut = async move {
-			// Events should only by generated
-			// if the response was successfully propagated.
+			let event = client
```

Comment:
move this below `rp_fut.await`? 

Makes not sense to me to do this storage call if  `rp_fut.await` produces an error...

Reply:
Yep, that makes sense! Thanks for catching this! 🙏